unittests.py

```diff
@@ -0,0 +1,33 @@
+import json
+import unittest
+import azure.functions as func
+from IngestRawXmlHttpTrigger.utils import get_url_from_req, OfsMissingUrlError
+
+
+class TestGetUrlFromReq(unittest.TestCase):
+    def setUp(self):
+        self.test_url = 'http://somehost.com/somepath'
+        self.valid_data = {
+            'resource_url': self.test_url}
+        self.invalid_data = {
```

Comment:
Worth extending test coverage with `empty_body`?

Reply:
Good point. Have added code for this.